modules/internal/ChapelTuple.chpl

```diff
@@ -112,7 +120,7 @@ module ChapelTuple {
 
   // last resort since if this resolves some other way, OK
   pragma "last resort"
-  proc *(p: int, type t) type {
+  proc *(p: integral, type t) type {
     compilerError("tuple size must be static");
```

Comment:
I don't think we use the word "static" in this sense in the docs/spec. Could you update the error to say "tuple size must be param" or "tuple size must be known at compile time"?

Reply:
I've changed this to your preferred wording.